azext_iot/central/commands_destination.py

```diff
@@ -0,0 +1,121 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# Dev note - think of this as a controller
+
+from typing import List, Union
+from azext_iot.common import utility
+from azext_iot.constants import CENTRAL_ENDPOINT
+from azext_iot.central.providers import CentralDestinationProvider
+from azext_iot.central.models.enum import ApiVersion
+from azext_iot.central.models.v1_1_preview import (
+    DestinationV1_1_preview,
+    WebhookDestinationV1_1_preview,
+    AdxDestinationV1_1_preview,
+)
+
+
+def get_dataExport_destination(
```

Comment:
Please do not mix typing styles - stick to using _'s and fix in other areas too

Reply:
Please change style to using _'s

Reply:
fixed